eng/jacoco-test-coverage/pom.xml

```diff
@@ -283,12 +283,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-administration</artifactId>
-      <version>4.4.0-beta.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-administration;current} -->
+      <version>4.4.0</version> <!-- {x-version-update;com.azure:azure-security-keyvault-administration;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-certificates</artifactId>
-      <version>4.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-certificates;current} -->
+      <version>4.5.6</version> <!-- {x-version-update;com.azure:azure-security-keyvault-certificates;current} -->
```

Comment:
should this also be a minor version upgrade? I saw other KV packages are upgraded with a minor version increase.

Reply:
There are no changes that warrant upgrading Certificates to a new minor version in this release.